main.go

```diff
@@ -154,3 +208,78 @@ func exitWithError(msg ...interface{}) {
 	fmt.Println(msg...)
 	os.Exit(1)
 }
+
+// generates relevant nethermind chain configuration files
+// logic ported over from audius-docker-compose https://github.com/AudiusProject/audius-docker-compose/blob/stage/audius-cli#L848
+func configureChainSpec() {
+	extraVanity := "0x22466c6578692069732061207468696e6722202d204166726900000000000000"
+	extraSeal := "0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
these were hardcoded in audius-cli 
https://github.com/AudiusProject/audius-docker-compose/blob/stage/audius-cli#L46-L47